frontend/javascripts/oxalis/view/rendering_utils.ts

```diff
@@ -106,6 +117,26 @@ export async function downloadScreenshot() {
     const buffer = renderToTexture(planeId, null, null, false, clearColor);
 
     const inputCatcherElement = document.querySelector(`#inputcatcher_${planeId}`);
+    const scalebar = inputCatcherElement?.querySelector(".scalebar") as HTMLElement;
+    let drawImageIntoCanvasCallback = null;
+    if (scalebar) {
```

Comment:
Since the scale bar and logo need to be toggled independently, I would propose to change the `right` and `bottom` position of the scale bar from the relative 1% to an absolute value like 10 px or so and add that to the constants. Then, the scalebar is no longer needed, here, and the position calculations are more straightforward.